### DISCUSSION TITLE
Run a shell command in split window like vscode?

### DISCUSSION THREAD

**Discussion by qiref** [helix-editor/helix#9351] [General]
<img width="902" alt="image" src="https://github.com/helix-editor/helix/assets/50668716/d30b0087-a537-4f92-aebc-d22e4f01d097">

I can use`:sh` In helix , but not convenient sometimes

**Comment by David-Else** [comment]
You will need to run a terminal multiplexer like Tmux/Zellij, or a terminal that has split windows like Kitty/Wezterm.
